index.js

```diff
@@ -7,12 +7,17 @@ let express = require('express');
 let mysql = require('./dbcon.js');
 let bodyParser = require('body-parser');
 let app = express();
+let session = require('express-session');
 const dotenv = require('dotenv');
 dotenv.config();
 
 // Handlebars
 let handlebars = require('express-handlebars').create({defaultLayout: 'main'});
 
+//mount session
+app.use(session({secret: 'secret', resave: true, saveUninitialized: true, signedin: false})); //store secret in .env
```

Comment:
storing secret in .env might be a bad idea in case we each have different secret keys, it should be in a central location, like in a shared database.